### PR TITLE
Update go sentry package from 0.8.0 to 0.10.0

### PR DESCRIPTION
**Pull request by miguelvr**
- [x] run `make test` and `make lint`
- [x] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,11 +1,9 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-cloud.google.com/go v0.38.0 h1:ROfEUZz+Gh5pa62DJWXSaonyu3StP6EA6lPEXPI6mCo=
```
Comment (RobertLucian) [inline go.sum:3]: robert: what's up with these removals here? Are these coming from the sentry update?

**Review by RobertLucian — APPROVED** [review]
LGTM!

File: go.sum (hunk 1 above)
Comment (miguelvr) [reply go.sum:3]: `go mod tidy`

Comment (RobertLucian) [reply go.sum:3]: Got it. Cool!

**Pull request merged**